### PR TITLE
world map: add remaining map icon info for issue #18488

### LINKED ISSUE #18488: Missing map icon information for Varlamore part 2
### Type

Other

### Operating System

None

### OS Version

_No response_

### Bug description

On the world map, when hovering over certain icons such as mining sites, fishing spots and map links, Runelite provides additional information such as the types of rock, type of fishing spot, or a description. These have not yet been added for map icons that were added with part 2 of Varlamore.

I've been through them all and checked what exactly they are, but I lack the knowledge to add the change myself or find the co-ordinates. After collating this information I did notice #18336 which has been open for a month and looks like it will add most of these, but it does not seem to include the dungeon information or the farming patch. (EDIT: I have struck through the information that is included in the referenced PR)

It'd be useful if this could get added before Leagues 5 starts, since Varlamore is one of the areas players can select and knowing about things like mining sites and fishing spots will be useful for any players who decide to pick Varlamore.

~~Fishing
Aldarin N - Small Net/Bait
Aldarin NW - Small Net/Bait
Aldarin S (Mistrock) - Cage/Harpoon
Varlamore N (Shipwreck Cove) SE - Big Net/Harpoon
Varlamore N (Shipwreck Cove) NW - Small Net/Bait
Salvager Overlook S (near Tower of Ascension) - Lure/Bait~~

~~Trees:
Aldarin NW - Magic
Aldarin NE - Yew
Mistrock N - Yew/Maple
Mistrock W - Maple
Mistrock E - Maple
Quetzacalli Gorge - Yew~~

~~Hunter: (note numbers are from the wiki, I didn't count them myself so can't verify)
Aldarin W - Ruby Harvest (9)
Aldarin N - Copper Longtail (5)
Mons Gratia - Cerulean Twitch (5), Sapphire Glacialis (7), Snowy Knight (6)~~

~~Mining:
Aldarin S (Mistrock) - 5 Gold, 5 Silver, 6 Coal, 8 Iron
Salvager Overlook S - 3 Mithril, 2 Adamantite, 7 Coal
Salvager Overlook E - 5 Gold, 5 Iron~~

~~Transportation:
Aldarin S (next to the windmill) - Quetzal
Aldarin NE N - Ship to Sunset Coast
Aldarin NE E - Charter Ship
Sunset Coast W -  Ship to Aldarin
Quetzacalli Gorge - Quetzal
Salvager Overlook - Quetzal~~

Farming Patches:
Aldarin - Hops

Dungeon map links:
Aldarin S (where the minigame icon is) - Alchemical Society
Aldarin SW - Moonrise Brewery and Winery
Aldarin Villa Lucens - Wine Cellar
Aldarin Villa Lucens East - Backstage
Varlamore W (east of Cam Torum and Perilous Moons quest) - Wolf Den

Dungeons:
Twilight Temple S - Ruins of Tapoyauik
Twilight Temple SW - Ruins of Tapoyauik

~~Agility Training:
Varlamore S - Colossal Wyrm Agility Course~~

~~Minigames:
Aldarin - Mastering Mixology~~

### Screenshots or videos

_No response_

### RuneLite version

Runelite version: 1.10.42.1
Launcher version: 2.6.7

### Logs

_No response_

### PR DESCRIPTION
**Pull request by Scrubmarine**
Close #18488 (and consequently #18336)

Added remaining map icon info requested in issue #18488

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/DungeonLocation.java [hunk 1]
```diff
@@ -31,8 +31,13 @@
 @Getter
 enum DungeonLocation
 {
+
```
Comment (Felanbird) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/DungeonLocation.java:34]:
```suggestion
```

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/DungeonLocation.java [hunk 2]
```diff
@@ -31,8 +31,13 @@
...
 	ABANDONED_MINE("Abandoned Mine", new WorldPoint(3439, 3232, 0)),
 	ABANDONED_MINE_SECRET("Abandoned Mine (secret entrance)", new WorldPoint(3452, 3244, 0)),
+	ALDARIN_WINE_CELLAR("Wine Cellar", new WorldPoint(1447, 2938, 0)),
+	ALDARIN_BACKSTAGE("Backstage", new WorldPoint(1475, 2927, 0)),
```
Comment (Felanbird) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/DungeonLocation.java:38]: The wiki calls it `Backstage entrance` which might be a tiny bit more descriptive? but idk

**Review by Felanbird — APPROVED** [review]
they all look correct to me

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/DungeonLocation.java (hunk 2 above)
Comment (Scrubmarine) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/DungeonLocation.java]: You're right, I was kind of just going off the name of the issue. Fixed it.

**Comment by Nightfirecat** [comment]
Added in c42b3f00d18d1bc094b595f2648ae399d8939326

**Pull request closed without merging**